### DISCUSSION TITLE
toLowerCase is not a function

### DISCUSSION THREAD

**Discussion by jcfinnerup** [colinhacks/zod#2123] [Q&A] [answered]
Hi there! 

Anyone else experiencing this?

## Code

```ts
await z.object({
    text: z.string().toLowerCase(), 
    another_text: z.string(), 
}).strict().safeParseAsync(_some_object)
```

## Expected behavior
No error thrown

## Actual Behavior
`TypeError: __vite_ssr_import_4__.z.string(...).toLowerCase is not a function`

**Comment by JacobWeisenburger** [comment]
I'm not sure why, but it seems that `toLowerCase` was in the docs, but it wasn't implemented. I'm not sure if it's going to be in the future. I have commented it out of the docs, and it can be easily added back in if it is implemented in the future. Sorry about the confusion

**Comment by JacobWeisenburger** [comment, marked as answer]
`toLowerCase` should be working now in: [v3.21.0](https://github.com/colinhacks/zod/releases/tag/v3.21.0)
